packages/plugin-ext/src/main/browser/comments/comment-thread-widget.tsx

```diff
@@ -498,6 +499,16 @@ export class ReviewComment<P extends ReviewComment.Props = ReviewComment.Props>
             </div>
         </div>;
     }
+    localeDate(stamp: string | undefined): string {
```

Comment:
minor: no need to abbreviate

```suggestion
    localeDate(timestamp: string | undefined): string {
```

Reply:
I now use timestamp rather than stamp in commit 376dbca.